docs/source/user.rst

```diff
@@ -32,15 +32,38 @@ Edit your ``conf.py`` file:
     ]
 
 3. At the bottom of ``conf.py``, if the following block of code is present,
-   remove it:
+   remove or modify it, otherwise ``sphinx_rtd_theme`` or the default readthedocs
```

Comment:
edit the code block to select the desired theme to use for any local builds (such as during development):

*block to look for here*


If you are building documentation locally, you may specify a desired theme (jupyter_alabaster_theme) or the sphinx_rtd_theme:

*jupyter_alabaster theme* - use #72 code

*sphinx_rtd_theme - show example below.